traffic_ops/app/lib/UI/ConfigFiles.pm

```diff
@@ -312,6 +312,9 @@ sub param_data {
 		if ( $row->parameter->name eq "location" ) {
```

Comment:
...before i lose this thought. does a config file typically have a variance of locations or scopes? for example, can the value of location and scope vary by profile?

profile 1:  file = cache.config, location = /opt/trafficserver/etc/trafficserver/, scope = server
profile 2: file = cache.config, location = /opt/somewhereelse/, scope = profile

if not, even more reason imo that a configfiles table may make sense. something to think about.